addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/discovery/MaxCubeBridgeDiscovery.java

```diff
@@ -65,32 +88,36 @@ public void startScan() {
 	 * @see org.eclipse.smarthome.config.discovery.AbstractDiscoveryService#
 	 * startBackgroundDiscovery()
 	 */
-
 	@Override
 	protected void startBackgroundDiscovery() {
-		discoverCube();
+		if (cubeBackgroundDiscoveryJob == null || cubeBackgroundDiscoveryJob.isCancelled()) {
+			cubeBackgroundDiscoveryJob = scheduler.scheduleAtFixedRate(cubeDiscoveryRunnable, 0, refreshInterval, TimeUnit.SECONDS);
+		}
 	}
 
 	@Override
 	public boolean isBackgroundDiscoveryEnabled() {
```

Comment:
You should actually not override this method as it is already implemented in the super class. The idea of it is that it does not say if anything is currently running in the background, but to say whether or not the framework should consider this service when activating background scans (and this can be enabled or disabled through the config admin.